build.gradle

```diff
@@ -16,31 +14,3 @@ buildscript {
 if (project.hasProperty('releasing')) {
     apply from: "release.gradle"
 }
```

Comment:
I think one approach here would be to apply `release.gradle` in both modules, and release each module as a separate artefact (with bugsnag-spring depending on bugsnag-java). It should be possible to update the script so that any module-specific values (e.g. package name) are supplied via `gradle.properties` or some equivalent mechanism.

In terms of testing this out I'd say we could mitigate the risk by:

- Triggering the task to generate a POM and manually inspecting it
- Uploading an alpha for both artefacts to JCenter/MavenCentral

Did you have this sort of approach in mind or something different?

Reply:
This was the approach that I had in mind.